src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -412,6 +412,7 @@ public void UrlDecode(string decoded, string encoded)
 
         [Theory]
         [MemberData(nameof(UrlDecodeData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework assumes the current codepage is UTF8")]
```

Comment:
I just included the comment on what @ianhays stated in the issue: https://github.com/dotnet/corefx/issues/13254#issuecomment-279495134

I just did a small investigation and this also fails in a 4.6.2 full framework console app. The inputs that are failing are the ones that have u0061 encoded which is correctly in UTF8:
* [failing input 1](https://github.com/dotnet/corefx/blob/master/src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs#L401)
* [Failing input 2](https://github.com/dotnet/corefx/blob/master/src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs#L402)